spec.md

```diff
@@ -52,20 +52,19 @@ A digital signature scheme is a fundamental cryptographic primitive that is used
 
 However traditional digital signatures are limited to fixed mode of signing and verifying, that is the entire payload that was signed by a signer must be known by the verifier in-order to validate the digital signature.
 
-This document describes the BBS signature scheme. The scheme features many important properties:
+This document describes the BBS signature scheme. (FIXME: THE BBS signature scheme? Or A BBS signature schemes? Also: BBS citation needed/informative reference).
+The scheme feature important properties that allow the scheme to be used in applications where privacy and data minimization techniques are desired and/or required:
 
-1. The signature is over a group of Pedersen commitments--signatures can be created blinded or un-blinded.
+1. Signatures can be created blinded or un-blinded.
    
-2. The signature is encoded as a single group element and two field elements.
+2. Signatures are encoded as a single group element and two field elements. (FIXME: Why is this important? Succinctness? Unclear)
```

Comment:
Why does it need justification?

Reply:
Because above it it says "The scheme features many important properties:"
It is unclear why this generic property is important.
I changed this sentence above in the pull request already.
If this is just a general note then it is not an important property of the scheme I think.

Reply:
Also, the "encoding" of the signature would usually refer to a normative (interoperable) wire format of the signature. But the encoding  Maybe what is meant here is that the signatures are "represented" as a single group element and two field elements?
Later in the text it is also stated that "How a signature is to be encoded is not covered by this document. (TODO perhaps add some additional information in the appendix)"

Reply:
Yeah I think this is covered by issue #2 and we should aim to review it on the next working group call to provide better clarity around the direction the WG would like to take this

Reply:
+1 reading the context it does not appear apparent why this needs to be bulleted as it is, I would vote to remove it.